src/meta/date.rs

```diff
@@ -38,7 +38,9 @@ impl Date {
             DateFlag::Date => self.0.format("%c").to_string(),
             DateFlag::Relative => format!("{}", HumanTime::from(self.0 - Local::now())),
             DateFlag::ISO => {
-                if self.0 > Local::now() - Duration::days(365) {
+                // 356.2425 * 24 * 60 * 60 = 31556952 seconds per year
```

Comment:
A tiny typo, other than that we are good to go.
```suggestion
                // 365.2425 * 24 * 60 * 60 = 31556952 seconds per year
```

Reply:
Also, rebase with the current master. There seems to be a conflict in the `CHANGELOG.md` file.

Reply:
Done